pkg/flagset/authbearer.go

```diff
@@ -111,6 +111,20 @@ func AuthBearerWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"REVA_OIDC_ID_CLAIM"},
 			Destination: &cfg.Reva.OIDC.IDClaim,
 		},
+		&cli.StringFlag{
+			Name: "oidc-uid-claim",
+			Value:       "cern_uid",
```

Comment:
leave empty or use  `uidnumber` (the ldap attribute) because `cern_uid` is a custom claim.

Reply:
Sorry about that! Leaving it empty as reva will then just skip checking for these